shared/chat/conversation/messages/wrapper/shared.js

```diff
@@ -12,20 +12,82 @@ const LeftMarker = ({author, isYou, isFollowing, isBroken}) => (
   <Box style={{..._leftMarkerStyle, backgroundColor: marginColor(author, isYou, isFollowing, isBroken)}} />
 )
 
-const UserAvatar = ({author, showImage}) => (
-  <Box style={_userAvatarStyle}>
-    {showImage && <Avatar size={24} username={author} skipBackground={true} />}
-  </Box>
-)
+type UserAvatarProps = {
+  author: string,
+  showImage: boolean,
+  onClick: (author: string) => void,
+}
+
+class UserAvatar extends PureComponent<void, UserAvatarProps, void> {
+  _onClick: () => void
+
+  constructor(props: UserAvatarProps) {
```

Comment:
I don't understand why it would be needed. you can just change the Box to a ClickableBox and have onClick={() => onUserClick(author)} and it should work fine